chrome-extension/src/pages/app/functions/chartJSData.ts

```diff
@@ -1,21 +1,27 @@
-import { ILengthsOfChartBars, IRelevantData } from "../../../types/types";
-
-export function getChartJSData(allRequestData: IRelevantData): ILengthsOfChartBars{
-  const output: ILengthsOfChartBars= [];
+import { IChartJSData, IRelevantData } from "../../../types/types";
 
+export function getChartJSData(allRequestData: IRelevantData): IChartJSData{
+  const ChartJSData: IChartJSData = {
+    barLengths: [],
+    labels: [],
+    backgroundColors: []
+  }
   allRequestData.forEach((request) => {
-    const singleChartJSData: number[] = [request.relativeStartTime, request.relativeStartTime + request.duration];
-    output.push(singleChartJSData)
-  })
+    const {relativeStartTime, duration, name, source} = request;
+    const barLength: number[] = [relativeStartTime, relativeStartTime + duration];
+    ChartJSData.barLengths.push(barLength)
 
-  return output;
-}
+    ChartJSData.labels.push(name);
 
-export function getChartJSLabels(allRequestData: IRelevantData):string[]{
-  const labelsArr:string[] = [];
-  allRequestData.forEach((request) => {
-    labelsArr.push(request.name);
+    let barColor: string;
+    if(source === "Server-side"){
```

Comment:
I think the "...-side" under the Source column may be unnecessary and less aesthetic. 

If we want to ensure there is no confused between "initiator" and "Rendering", we could potentially create a column label with "Rendering" and the values could be "CSR" or "SSR". Or we can make the values "Client" and "Server". 

The column label itself maybe confusing as is, because "Source" can be considered a synonym of "Initiator" and devs may be used to seeing "Initiator" on the original devtools

Reply:
Yes I think that change would make it clearer. Just updated!